### ISSUE TITLE
Colour contrast of mobile menu when magnified

### ISSUE THREAD

**Issue by stef-barburou** [snap-framework/snap#17]
Tool(s): colour contrast analyser, visual check

Browser/Screen reader: firefox

WCAG 2.0 reference: 1.4.3 Contrast (minimum)

Details: Using the colour contrast analyser we found that when the page is magnified to 200% the mobile menu fails colour contrast.

![image](https://user-images.githubusercontent.com/41083520/74272285-6210b680-4cdc-11ea-9fe6-6638bd7a98b3.png)
